boards/others/promicro_nrf52840/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_PROMICRO_NRF52840
+
+config BOARD_HAS_NRF5_BOOTLOADER
+	bool "Board has nRF5 bootloader"
+	default y
+	help
+	  If selected, applications are linked so that they can be loaded by Nordic
+	  nRF5 bootloader.
+
+config BOARD_SERIAL_BACKEND_CDC_ACM
+	bool "USB CDC"
+	default y
```

Comment:
use common configuration - see https://github.com/zephyrproject-rtos/zephyr/pull/81308/commits/a233b2d43fb14ec9558b01aa4ea859f0a1b281e8

Reply:
Done.

Reply:
this Kconfig needs to be removed, the defconfig is the right one